pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -547,26 +548,33 @@ func (ds *Plugin) openConnection(config *configuration, isReadOnly bool) error {
 	}
 
 	if sqlDb == nil || connectionString != sqlDb.connectionString || config.DatabaseType != ds.db.databaseType {
-		db, err := ds.openDB(config, isReadOnly)
+		db, version, supportsCTE, err := ds.openDB(config, isReadOnly)
 		if err != nil {
 			return err
 		}
 
-		if sqlDb != nil {
-			sqlDb.Close()
-		}
-
 		raw := db.DB()
 		if raw == nil {
 			return sqlError.New("unable to get raw database object")
 		}
 
+		if sqlDb != nil {
+			sqlDb.Close()
+		}
+
+		ds.log.Info("Opened SQL database",
```

Comment:
Curious to hear a little bit about the motivation for this new log line. This isn't a change I would have expected to impact log emission.

Reply:
I thought it might be nice for debuggability. If somebody hits a problem with the SQL plugin, it'd be nice for the version to show up in the logs w/o having to ask them.